sp_api/base/marketplaces.py

```diff
@@ -30,15 +30,15 @@
 import os
 
 
-class AWS_ENV(Enum):
+class AwsEnv(Enum):
     PRODUCTION = "PRODUCTION"
     SANDBOX = "SANDBOX"
 
 
-AWS_ENVIRONMENT = os.getenv("AWS_ENV", AWS_ENV.PRODUCTION.name)
+AWS_ENVIRONMENT = os.getenv("AWS_ENV", AwsEnv.PRODUCTION.name)
 BASE_URL = "https://sellingpartnerapi"
 
-if AWS_ENV(AWS_ENVIRONMENT) is AWS_ENV.SANDBOX:
+if AwsEnv(AWS_ENVIRONMENT) is AwsEnv.SANDBOX:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `is` operator performs an identity comparison that uses the built-in `is()` method to test if the compared items are the same object or  not. It should be used with only singletons, such as `None`. We recommend that you use the `==` or `!=` equality operators to determine if two objects are equal.

[Learn more](https://docs.python.org/3/reference/datamodel.html#object.eq)